extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -264,20 +264,46 @@ export class SchemaCompareResult {
 		return data;
 	}
 
+	private getFormattedScript(diffEntry: azdata.DiffEntry, getSourceScript: boolean): string {
+		let script = '';
+		if (getSourceScript) {
+			script = diffEntry.sourceScript === null ? '\n' : this.getAggregatedScript(diffEntry, getSourceScript);
+		} else {
+			script = diffEntry.targetScript === null ? '\n' : this.getAggregatedScript(diffEntry, getSourceScript);
+		}
+
+		// add new lines after 'GO's
+		script = script.replace(new RegExp('GO', 'g'), 'GO\n');
+
+		// the script needs to end with \n\n otherwise it shows as a difference even if there isn't anything
+		if (!script.endsWith('\n\n') && script !== '\n') {
+			script += script.endsWith('\n') ? '\n' : '\n\n';
+		}
+
+		return script;
+	}
+
 	private getAggregatedScript(diffEntry: azdata.DiffEntry, getSourceScript: boolean): string {
 		let script = '';
 		if (diffEntry !== null) {
 			script += getSourceScript ? diffEntry.sourceScript : diffEntry.targetScript;
+			if (this.isValidScript(script)) {
+				script = script.trim() + '\nGO';
```

Comment:
definitely looks prettier - but not sure if we should add/subtract anything in the script from here. @pensivebrian  what do you think - is this okay to do?